ZachDAmico/reader-client
# add form done

I have the form for adding a new book completed

to Test,
go to url
input text into each field - checking state in components tab of dev tools
check category boxes - checking state in components tab of dev tools
all state should match form input text/choices
